### PR TITLE
Document the stabilized BitOps versioning in the docs

### PR DESCRIPTION
**Pull request by armab**
Following the collective effort in the discussions (https://github.com/bitovi/bitops/discussions/203), WIKI and CD work, document the stabilized official BitOps versioning, showing an example with a table and explanation on how images are named and tagged in different scenarios.

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -37,9 +37,10 @@ nav:
     - Default Environment: default-environment.md
   - Examples: examples.md
   - Upgrade:
-    - 2.0.0: migration/2.0.0-migration.md
+    - 2.0: migration/2.0-migration.md
   - Development:
     - Local development: development-local.md
+    - Versioning: versioning.md
     - Custom Images: custom-image.md
```
Comment (armab) [inline mkdocs.yml:44]: For the category, the page is under `Development`, but I'm open to suggestions if anyone has a better idea where to place the `Versioning` page.

Comment (ccapell) [reply mkdocs.yml:44]: IMO, I would put it under configuration.  Development to me is more about customizing an image.

**Review by ccapell — APPROVED** [review]

File: mkdocs.yml (hunk 1 above)
Comment (ccapell) [reply mkdocs.yml:44]: Just a thought.

Comment (armab) [reply mkdocs.yml]: The "Configuration" section would work better indeed.
Updated in https://github.com/bitovi/bitops/pull/278/commits/50e358b2ebba2f7429d4df252dce8d43aa1e0bef, thanks!

**Pull request merged**